examples/python/trigger-pagerduty-incident/handler/handler.py

```diff
@@ -1,11 +1,30 @@
 import sys, json, os
 import urllib3
 import requests
-import json
+import traceback
+
+# GLOBAL_VARS
+DEBUG=False
+class bgc:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
 
-# Surpress SSL warnings
 if(os.getenv("insecure_ssl")):
+    # Surpress SSL warnings
     urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+if(os.getenv("write_debug")):
+    sys.stderr.write(f"{bgc.WARNING}WARNING!! DEBUG has been enabled for this function. Sensitive information could be printed to sysout{bgc.ENDC} \n ")
+    DEBUG=True
+
+def debug(s):
+    if DEBUG:
+        sys.stderr.write(s+" \n ") #Syserr only get logged on the console logs
```

Comment:
Did you verify this just locally or also in a fully working VEBA environment? Asking because I'm not sure how this is interpreted in Kubernetes/OpenFaaS.

Reply:
this was verified on OpenFaaS (VEBA v0.3)